client/src/+assets/user-publish-steps/main-page.tsx

```diff
@@ -190,14 +196,18 @@ export const UserPublishMultiStep: NextPage = () => {
             onAssetPublish({nftAddress: getNftTierAddress(), metadata: generateMetadata()})
```

Comment:
better to try to use async await, easier to read

Reply:
in this specific case I prefer to use Promise syntax, is easier to handle the specific result of the call compare to await with try/catch